src/GitHub.VisualStudio/Styles/VsBrush.xaml

```diff
@@ -10,4 +10,6 @@
   <SolidColorBrush x:Key="GitHubVsCommandBarSelectedBorder" Color="{DynamicResource VsColor.CommandBarSelectedBorder}" />
   <SolidColorBrush x:Key="GitHubVsSearchBoxBackground" Color="{DynamicResource VsColor.SearchBoxBackground}" />
   <SolidColorBrush x:Key="GitHubVsWindowText" Color="{DynamicResource VsColor.WindowText}" />
+  <SolidColorBrush x:Key="GitHubVsTextBoxBorder" Color="{DynamicResource VsColor.BrandedUIBorder}" />
+  <SolidColorBrush x:Key="GitHubVsTextBoxBackground" Color="{DynamicResource VsColor.BrandedUIBackground}" />
```

Comment:
For brushes from VS, make sure the names match whatever property name you're using in VsColor, so this should be `GitHubVsBrandedUIBorder` instead.